Detector/DetSensitive/DetSensitive/SimpleCalorimeterSDWithBirksLaw.h

```diff
@@ -0,0 +1,62 @@
+#ifndef DETSENSITIVE_SIMPLECALORIMETERSD_WITHBIRKS_H
```

Comment:
This should be DETSENSITIVE_FILENAME as described in [guidelines](https://github.com/HEP-FCC/FCCSW/blob/master/doc/CppCodingStyleGuidelines.md#include-guard)